internal/controller/core/testdata/secrets/dataconnection-string-http.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
 stringData:
   AWS_ACCESS_KEY_ID: admin
   AWS_DEFAULT_REGION: us-south
-  AWS_S3_BUCKET: modelmesh-example-models
```

Comment:
This is just a bucket name, so it should be kept for e2e tests

Reply:
Does it need to have "modelmesh" in its name?